### DISCUSSION TITLE
Region Prompt Control

### DISCUSSION THREAD

**Discussion by iron-marshal** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#77] [Q&A]
I downloaded into Automatic1111 any was trying to follow through the examples but I don't see anywhere to enable Region Prompt Control. Where/how can I enable it?

Thanks.

**Comment by pkuliyi2015** [comment]
Would you please give us a screen shot on what's your extension look like?

**Reply by iron-marshal** [reply]
Thanks for any help you can provide. I've attached a screen shot of the txt2img tab as well as my settings.
![Multidiffusion VAE](https://user-images.githubusercontent.com/58190738/228389345-a0f22c88-f83d-4cdf-aefe-5dabd24dbf8b.png)
![settings](https://user-images.githubusercontent.com/58190738/228389368-d58a9089-8dd4-4e86-a6d8-b962d9a4ba5a.png)

**Comment by pkuliyi2015** [comment]
Well, it is exactly inside your picture. Look at here:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/59076257/228390448-090cd469-be25-4203-b3ce-0b391a3cd9f9.png">
Expand the accordion and you will see everything.

**Reply by ToniMagi** [reply]
In my case it doesn't look like this:

![multidifussion](https://user-images.githubusercontent.com/129340123/228875614-5600615e-2862-4533-99a1-146c9da91779.PNG)

The options indicated in the tutorial do not really appear in my interface. I've updated the extension, but the interface doesn't change or new options appear. TIA for the help!

**Reply by ToniMagi** [reply]
How odd. I have deleted from the 'Extensions' folder some extensions that I already had deactivated in the UI, I have updated the ones that are activated and now the complete interface appears, with the 'Region Prompt Control' option. Problem solved.
